### ISSUE TITLE
Density in the centre of reconstructed box - CTF issue?

### ISSUE THREAD

**Issue by Harsh-Ranawat** [ml-struct-bio/cryodrgn#391]
Hello and thanks for this very useful tool! 
I have particles exported from cryosparc, downsampled in cryodrgn from 440px to 128px (input particle stack is the .cs file from cryosparc where the CTF and poses are also sourced from)
After training I see a concentric density in the centre of the box 
Any idea where this comes from and how it can be troubleshooted?

attached a video that traverses PC1 👇🏽
https://github.com/user-attachments/assets/c232e488-3e5a-4192-910d-6466a1c5d662

**Comment by michal-g** [comment]
Hi, can you please share the command (and parameter settings) that you used when you got this output volume? Do you see the same problem when try different values for parameters such as learning rate, latent space dimension, etc.?

**Comment by Harsh-Ranawat** [comment]
the training command I used for the run in question:
cryodrgn train_vae particles.mrc --ctf ctf.pkl --poses poses.pkl --max-threads 128 --zdim 8 -n 50 --enc-dim 256 --enc-layers 3 --dec-dim 256 --dec-layers 3 --lazy --multigpu -o outputs> out.log

I have tried this for a network with --enc-dim 128 and --dec-dim 128 with the same artefact.
I haven't tried tweaking other parameters as of yet, though.

Any first suggestions?

**Comment by michal-g** [comment]
Have you checked the UMAP clustering of the latent space (`analyze.49/umap_hexbin.png`) and the volumes reconstructed across it (`analyze.49/kmeans20/vol_*.mrc`) to see if this artefact is perhaps associated with an identifiable subset of the particles that may be junk?

The first parameter I would try adjusting next is the model learning rate (`--lr`); default is 1e-4, but maybe try 1e-5, 5e-4, 1e-3, etc.

Especially if the rest of the volume is coming out more or less how you would expect (even with just using backprojection), you may simply have to use a particle stack only downsampled as far as 256px, or an even larger model (`enc-dim` and `dec-dim` of 512 or 1024) in order to properly resolve what is happening in the center of the volume.

Also, if you have only been analyzing the final 50th epoch, you can also check earlier epochs to see if perhaps this isn't an artefact of the model overfitting after too many iterations. This is a good idea in any case to get a feel for the convergence (or lack thereof) in the model.

Let me know if that helps!
